src/components/Metrics.tsx

```diff
@@ -14,24 +14,114 @@
  * limitations under the License.
  ***************************************************************************** */
 
+import { reaction } from 'mobx';
+import { observer } from 'mobx-react-lite';
+import { useEffect, useMemo, useState } from 'react';
 import { createUseStyles } from 'react-jss';
+import { useBoxesStore } from '../hooks/useBoxesStore';
+import { useInput } from '../hooks/useInput';
+import { useSchemaStore } from '../hooks/useSchemaStore';
+import Input from './util/Input';
 
 const useStyles = createUseStyles({
 	container: {
 		border: '1px solid',
 		gridArea: 'metrics',
 		borderRadius: 6,
+		overflow: 'hidden',
+		display: 'flex',
+		flexDirection: 'column',
+		justifyContent: 'space-between',
+	},
+	metricsSection: {
+		height: '33%',
+	},
+	metrics: {
+		height: '100%',
+		width: '100%',
+		border: 'none',
+	},
+	logs: {
+		height: 'calc(100% - 40px)',
+		width: '100%',
+		border: 'none',
 	},
 });
 
 function Metrics() {
 	const classes = useStyles();
+	const schemaStore = useSchemaStore();
+	const boxesStore = useBoxesStore();
+
+	const search = useInput({
+		initialValue: '',
+		id: 'search',
+		label: 'Search',
+	});
+
+	const [component, setComponent] = useState<string>('');
+
+	const options = useMemo(() => {
```

Comment:
since options inly uses observable data that makes more sense to use computed here
```
const options = computed(() => `var-namespace=th2-${schemaStore.selectedSchema}&theme=light`).get()
```